app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, current_file, copied_file = command.split()
+    if cp == "cp" and copied_file != current_file:
+        with open(f"{current_file}", "r") as file_in, \
+             open(f"{copied_file}", "w") as file_out:
```

Comment:
better to use `()` then `\` 
```suggestion
        with (
            open(f"{current_file}", "r") as file_in, 
            open(f"{copied_file}", "w") as file_out
        ):
```